src/views/mainView.py

```diff
@@ -90,6 +92,13 @@ def update(self, settings, sensordata):
         self.oxy_btn.setText("Oxygen (02)" + '\n' + str(self.settings.oxygen) + " [%]")
         self.oxy_btn.setBackground(self.getBtnColor(AlarmType.OXYGEN_TOO_LOW, AlarmType.OXYGEN_TOO_HIGH))
 
+        self.inspHold_btn.setEnabled(settings.start)
+        if sensordata.inspiratory_hold_result:
+            self.inspHold_btn.setText("Inspiration Hold\n" + str(sensordata.inspiratory_hold_result) + " [cm H2O]")
```

Comment:
Ik denk dat het beter is om die button text zo te formatten:
```
'Inspiration Hold\n{0:.2g} [cm H2O]'.format(sensordata.inspiratory_hold_result)
```
Anders krijg je daar `20.1234567789124` zulke getallen te zien